components/scream/src/control/surface_coupling_export.cpp

```diff
@@ -59,16 +64,23 @@ void SurfaceCoupling::do_export (const bool init_phase)
       PF::calculate_dz(team, pseudo_density_i, p_mid_i, T_mid_i, qv_i, dz_i);
       team.team_barrier();
 
-      // Compute vertical layer heights. Use z_int(nlevs) = z_surf = 0.0.
+      // Compute vertical layer heights (relative to ground surface rather than from sea level).
+      // Use z_int(nlevs) = z_surf = 0.0.
       const Real z_surf = 0.0;
       PF::calculate_z_int(team, num_levs, dz_i, z_surf, z_int_i);
       team.team_barrier();
       PF::calculate_z_mid(team, num_levs, z_int_i, z_mid_i);
       team.team_barrier();
 
+      // Calculate air temperature at bottom of cell closest to the ground for PSL
+      // ?would be better to do this in a function so it can be better tested?
```

Comment:
I'm not particularly comfortable with getting surface T by extrapolating the 2 lowest midpoint points. EAMf90 computes this by assuming a lapse rate of 6.5 K/km, which is perhaps more numerically stable but is otherwise way more terrible. Assuming we want to use this formulation, should we be interpolating in Z or in pressure? Should we use column_scan or create a standalone function to do this computation rather than hardcoding it in here (where it isn't well tested)?